src/main/java/de/tum/in/www1/artemis/service/connectors/GitService.java

```diff
@@ -639,10 +640,20 @@ public Repository getExistingCheckedOutRepositoryByLocalPath(@NotNull Path local
      */
     public void commit(Repository repo, String message) throws GitAPIException {
         try (Git git = new Git(repo)) {
-            git.commit().setMessage(message).setAllowEmpty(true).setCommitter(artemisGitName, artemisGitEmail).call();
+            commit(git).setMessage(message).setAllowEmpty(true).setCommitter(artemisGitName, artemisGitEmail).call();
         }
     }
 
+    /**
+     * Creates a CommitCommand and sets signing to false.
```

Comment:
Can you extend the comment to include a reason for not signing a commit and/or explaining what the effect of changing setSign to false will have?

Reply:
Good Idea, done!